Samples for direct debugging from a pod (#27382)
kubernetes/website: content/en/docs/tasks/debug-application-cluster/debug-service.md

```none
kubectl run -it --rm --restart=Never busybox --image=gcr.io/google-containers/busybox sh
```
{{< note >}}
If you don't see a command prompt, try pressing enter.
{{< /note >}}

> Alternatively you can run the command directly from the pod. For example, let us say we wish to run nslookup for an existing service `servicesample` and store the output in nslookupout.txt. Then the command will be 

Reviewer: I'd write:
```suggestion
Alternatively you can run the command directly from the pod. For example, let us say you wish to run `nslookup `for an existing Service named `servicesample`, and will store the output in `nslookupout.txt`. A command that achieves this is:
```

> ```none

Reviewer: nit:
this is valid shell; we could highlight it as such

Reviewer 2: ```suggestion
```sh
```

kubectl run test-nslookup --image=busybox:1.28 --rm -it --restart=Never  -- nslookup servicesample > nslookupout.txt
```

> Suppose you want to perform nslookup against an existing pod itself (say the pod ip is 10.244.1.13 in the default namespace), then the command would be 

Reviewer: ```suggestion
Suppose you want to perform nslookup against an existing pod itself. If the example pod's IPv4 address is 10.244.1.13, and the pod is in the `default` namespace, then you run:
```

Author: @sftim Thanks for the review Tim. the nslookup works only from versions starting 1.28 only. That is the reason, I used that version. 

Thanks for the corrections and feedback.

```none
kubectl run test-nslookup --image=busybox:1.28 --rm -it --restart=Never  -- nslookup 10-244-1-13.default.pod > nslookupout.txt
```

> Note that in this case, we have provided the automatically assigned service name of the pod with ip address specified with a '-' instead of '.'. 

Reviewer: ```suggestion
Notice that in this case, you have queried for the automatically assigned
hostname of the pod (based on its IPv4 address, but separated with a
`-` instead of `.`).
```

Reviewer 3: 👋 @ganeshniyer . Thanks for this contribution.
What do you think about accepting this suggestion?
Thanks.
